cli/tunnel/config.go

```diff
@@ -21,6 +22,7 @@ const (
 	DefaultBackoffMultiplier  = 1.5
 	DefaultBackoffMaxInterval = 60 * time.Second
 	DefaultBackoffMaxTime     = 15 * time.Minute
+	ConfigFileSTDIN           = "-"
```

Comment:
Seems like it's not part of "Default backoff configuration."

Reply:
This is only stdin file path reference.